gatsbyjs/src/utils/storyblok-service.js

```diff
@@ -1,84 +1,20 @@
 import { useEffect, useState } from "react"
-import StoryblokClient from "storyblok-js-client";
-import config from '../../gatsby-config'
-const sbConfig = config.plugins.find((item) => item.resolve === 'gatsby-source-storyblok')
+import { useStoryblokBridge } from "@storyblok/js";
```

Comment:
```suggestion
import { useStoryblokBridge } from "@storyblok/react";
```

Reply:
@fgiuliani As Gatsby will need a workaround for parsing `content` , I need to use `useStoryblokBridge` from JS SDK, not from React SDK.

Reply:
what's the difference between them? As far as I can see, the `useStoryblokBridge` function from `@storyblok/react` is the same as the one from `@storyblok/js`:
https://github.com/storyblok/storyblok-react/blob/feature/v1/lib/index.js#L11

Reply:
Same as the comment in pakcgae.json. Fixed and I'll push the fixes now. Thanks @fgiuliani !